src/compiler/compile/nodes/Attribute.ts

```diff
@@ -96,7 +96,7 @@ export default class Attribute extends Node {
 	}
 
 	get_static_value() {
-		if (this.is_spread || this.dependencies.size > 0) return null;
+		if (this.is_spread || !this.is_static) return null;
```

Comment:
```suggestion
		if (!this.is_static) return null;
```

If it's spread, it's not static